RELEASE_NOTES.md

```diff
@@ -27,240 +27,185 @@ will not be possible to downgrade from 3.0 to 2.6.x any more.
-  rotation to avoid the gradient to flip sporadically.
-
-- Make sure the OpenCL kernel code are recompiled when the driver
+- Make sure the OpenCL kernel code is recompiled when the driver
   version is updated.
 
-- Add color pickers in the split toning module.
-
-- Add color pickers in the graduatednd module.
-
-- Add color picker to the watermark module.
+- Add color pickers for 'split toning', 'graduated density' and 'watermark' modules.
 
-- The color picker point positions are kept during the editing on the
-  module.
+- The color picker positions are kept during editing within a module.
 
-- The map view will zoom on the images of the selected collection.
+- The map view can zoom on the images of the selected collection.
 
-- The slideshow supports changing the delay between each images.
+- The slideshow will now start at the selected images if any, and supports
+  changing the delay between images. It will also now be more responsive when
+  manually moving backward and forward through the images.
 
-- The slideshow will now start at the selected pictures if any.
-
-- The slideshow will now be more responsive when changing the picture
-  backward and forward.
-
-- A new module 'basic adjustments' has been added. It allows to adjust
+- A new 'basic adjustments' module has been added. It allows to adjust
   the exposure, highlight compression, contrast, brightness and saturation.
   It also has an auto feature based on Rawtherapee's auto levels that can
   work on the entire image or a user selected area.
-  Both the contrast and the auto feature use the middle grey point, the
-  default for this setting is based on the work profile.
-
-- A new module 'rgb curve' has been added. It has linked and
-  independent RGB channels.
-  It works with the rgb color space selected on the work profile.
-  The curves and histogram can be displayed using the work profile or
-  can be scaled to be 50% grey.
-  The color picker allows to select by area with <kbd>Ctrl+click</kbd>.
-  The second color picker add 4 nodes to the curve based on the area
-  selected from the image: min, average, middle, max.
-  When pressing the shortcut 'pan&zoom while editing masks' the draw area
-  can be zoomed.
-  Different types of curves can be selected.
-
-- A new module 'rgb levels' has been added. It has linked and
-  independent channels, preserve colors option and is native RGB.
 
-- A search text has been added to the module groups in the
-  darkroom. It has a configuration option to show only the modules
-  groups, only the search text or both.  Modules are searched by name
-  (localized).
+- A new 'rgb curve' module has been added. It has modes for linked and
+  independent RGB channels. The curves and histogram can be displayed using
+  the working profile or can be scaled to be 50% grey. A secondary color
+  picker will add 4 nodes to the curve based on the area selected in the
+  image: min, average, middle, and max.
```

Comment:
4 nodes only possible in RGB linked.

Reply:
I've only tried this just now, but it seems to do similar things on independent channels as on linked mode... in both cases, it can sometimes appear to be less than four nodes, if one or two are very close to corners. Is the 50% gray part worth explaining at length here, or could it be left out? Additional feedback welcome.

Reply:
You are correct, my bad, I just didn't sport the nodes. Sorry for wasting your time.

I will let you guys decide on 50% , as a new user I don't know what that could mean.